contrib/cirrus/integration_test.sh

```diff
@@ -37,11 +37,7 @@ case "$SPECIALMODE" in
                 -o CheckHostIP=no $GOSRC/$SCRIPT_BASE/rootless_test.sh ${TESTSUITE}
         ;;
     cgroupv2)
-        # FIXME: use the package once all the fixes are in a release
-        # yum install -y crun
-        setenforce 0
-        yum builddep -y crun
-        (git clone --depth=1 https://github.com/containers/crun && cd crun && ./autogen.sh && ./configure --prefix=/usr && make -j4 && make install)
+        yum install -y crun
```

Comment:
Should be `dnf` :smiley:

Reply:
errr...actually, I think we've already got crun installed in the VM image, no?   Maybe this should be a 'dnf update -y crun'?

Reply:
oh nope, that's the kernel stuff. nvm, 'install' is correct.